.changeset/popular-mirrors-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": minor
+---
+
+Remove license types `micro` and `subscription` as options from selection in `FileSettingsFields`.
```

Comment:
```suggestion
Remove license types `MICRO` and `SUBSCRIPTION` as options from selection in `FileSettingsFields`.
```

(since we also write it uppercase in https://github.com/vivid-planet/comet/pull/1531/files)